Sources/Testing/Running/Configuration+EventHandling.swift

```diff
@@ -13,12 +13,13 @@ extension Configuration {
   ///
   /// - Parameters:
   ///   - event: The event to handle.
+  ///   - context: The context associated with the event.
   ///
   /// This method is used to handle events which are generated by the testing
   /// library. Typically this only involves passing the event to this instance's
   /// `eventHandler` but this method may also be used as a customization point
   /// to change how the event is passed to the event handler.
-  func handleEvent(_ event: Event) {
-    eventHandler(event)
+  func handleEvent(_ event: consuming Event, in context: consuming Event.Context) {
```

Comment:
We might end up calling multiple event handlers (e.g. if an event occurs on a detached task) so these should be borrowing, not consuming, if anything.

Reply:
Oddly, the compiler isn't catching this (shouldn't it?)

Reply:
Since the type is copyable, I am pretty sure it just ends up copying it. i.e. `consuming` in parameter position doesn't really have any effect on copyable types. I think.